### PR TITLE
[ADD] added the website_portal_address

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis status

**Comment by JayVora-SerpentCS** [comment]
Could not face the error on local! Any guidance?

**Comment by pedrobaeza** [comment]
The error is due to `sass` command is not available. Let me check how to put it.

**Comment by lasley** [comment]
@pedrobaeza - I'm rebuilding all the recent failed builds from website now that the SASS thing is fixed.

**Comment by pedrobaeza** [comment]
But the PR should be rebased I think, as the .travi.yml file used is the one on the PR

**Comment by lasley** [comment]
Travis is awesome and does an auto-merge with the base for us before test, so we should actually be fine 😄

**Comment by lasley** [comment]
@pedrobaeza Bah ok I should have known that you were going to be right on something like this, because you totally are. 

@JayVora-SerpentCS please rebase on current 9.0 to get the fix for the SASS error.

**Comment by tafaRU** [comment]
@JayVora-SerpentCS, please see https://github.com/OCA/website/pull/273#issuecomment-263009656 (previous comment).

Thank you in advance.

**Comment by JayVora-SerpentCS** [comment]
Processed, not squashing in 1 commit as it has many commits since nov 09,2016.

**Comment by lasley** [comment]
@JayVora-SerpentCS - None of these commit notes mean anything to me. If you would like to make the case that they should all be kept, please make them all fit OCA contribution guidelines for commit text.

**Comment by JayVora-SerpentCS** [comment]
@lasley Sorry I did not get it well. The only commit of 'Added the module website_portal_address' will do.
Guide me if I misunderstood.
Thanks.

**Comment by lasley** [comment]
@JayVora-SerpentCS - There are numerous merge commits that do not add value to our changelog (18b8cff, 6f8275e, c867f9b, ...). We are asking that you either squash those commits appropriately into the logical change commits, or update the commit message to fall in line [with OCA guidelines on commit messages](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#commit-message).

The alternative is that we squash everything on merge, which will screw up authorship.

**Comment by JayVora-SerpentCS** [comment]
@lasley Squashed into one logical commented commit. Let me know if I still miss anything. Thanks.

File: website_portal_address/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2016-Today Serpent Consulting Services PVT. LTD. (http://www.serpentcs.com)
+# Copyright 2016-Today Serpent Consulting Services PVT. LTD.
+# (http://www.serpentcs.com)
```
Comment (lasley) [inline website_portal_address/__init__.py]: Why editing all these lines to make two lines? IMO this looks dirty & is out of spec for the OCA header.

Comment (JayVora-SerpentCS) [reply website_portal_address/__init__.py]: Made one-liner, thanks.

**Review by lasley — APPROVED** [review]

**Review by tafaRU — CHANGES_REQUESTED** [review]
Please replace Serpent icon by generic OCA icon.
The rest LGTM

**Comment by JayVora-SerpentCS** [comment]
Done.

**Review by tafaRU — APPROVED** [review]

**Pull request merged**